release_notes/topics/12_0_0.adoc

```diff
@@ -41,6 +41,13 @@ There are also design changes in login theme for better user experience.
 You can even define an icon for your custom Identity providers.
 For details, please refer to the link:{developerguide_link}#custom-identity-providers-icons[docs].
 
+== Default roles processing improvement
+
+Default roles are now internally stored as composite roles of a new role usually named 'default-roles'. Instead of assigning 
+both realm and all client default roles directly to newly created users or users imported through Identity Brokering, just the role is 
+assigned to them and the rest of default roles are assigned as effective roles. This change improves performance of default roles processing,
+especially with larger number of clients, because it's not necessary go through all clients any more.
```

Comment:
Change "it's not necessary go through all clients any more."
to
"it is no longer necessary to go through all clients."